consent-ws/src/main/java/org/broadinstitute/consent/http/service/DatabaseDataAccessRequestAPI.java

```diff
@@ -384,6 +397,18 @@ private void updateElection(Election access, Election rp) {
         }
     }
 
+    @Override
+    public byte[] createDARDocument(Document dar, Map<String, String> researcherProperties) throws NotFoundException, IOException {
+        ByteArrayOutputStream output = new ByteArrayOutputStream();
+        ClassLoader classLoader = getClass().getClassLoader();
+        InputStream is = classLoader.getResourceAsStream(PATH);
+        PDDocument darDOC = PDDocument.load(is);
+        dataAccessParser.fillDARForm(dar, researcherProperties, darDOC.getDocumentCatalog().getAcroForm());
+        darDOC.save(output);
+        output.close();
+        darDOC.close();
```

Comment:
typically files are closed in a finally block in case an exception is thrown during processing